C++ getting-started guide + better minimal example (#3989)
rerun-io/rerun: docs/content/getting-started/installing-viewer.md

---
title: Installing the Rerun Viewer
order: 0
---

> The [Rerun Viewer](../reference/viewer/overview.md) can be installed independent of the SDK language you're using.

Reviewer: I think we should make a note here that the viewer and SDK versions need to match.

> There's several ways to install it, pick whatever works best for your setup:

Reviewer: ```suggestion
There are many ways to install the viewer. Please pick whatever works best for your setup:
```
